tests/unit-fail/j041d-link1.ref

```diff
@@ -1,13 +0,0 @@
-
```

Comment:
oh never mind, looks like you are deleting them, git just did something really weird with the diff and thought they were renames. Okay, you can ignore this comment.